modules/electrophysiology_browser/jsx/electrophysiologyBrowserIndex.js

```diff
@@ -142,6 +142,14 @@ class ElectrophysiologyBrowserIndex extends Component {
       {label: 'Acquisition Time', show: true},
       {label: 'Insertion Time', show: true},
       {label: 'Links', show: true},
+      {label: 'Type', show: false, filter: {
+        name: 'type',
+        type: 'select',
+        options: {
+          Raw: 'Raw',
```

Comment:
Might be more flexible to pass the options as props to stay consistent with the database state. (see how this is done for Site, Project).

Reply:
Good point!